### PR TITLE
OCPBUGS#21817: Updates prereq for customLogoFile size

### PR DESCRIPTION
**Pull request by opayne1**
OCPBUGS#21817: Updates prereq for customLogoFile size

Version(s):
4.12, 4.13, & 4.14

Issue:
https://issues.redhat.com/browse/OCPBUGS-21817

Link to docs preview:
https://66695--docspreview.netlify.app/openshift-enterprise/latest/web_console/customizing-the-web-console#adding-a-custom-logo_customizing-web-console

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by opayne1** [comment]
Hi @ocasalsa Could you take a look at this PR to update the `customLogoFile` size? thank you!

**Comment by yanpzhan** [comment]
The update LGTM.

File: modules/adding-a-custom-logo.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ You can create custom branding by adding a custom logo or custom product name. Y
 .Prerequisites
 
 * You must have administrator privileges.
-* Create a file of the logo that you want to use. The logo can be a file in any common image format, including GIF, JPG, PNG, or SVG, and is constrained to a `max-height` of `60px`.
+* Create a file of the logo that you want to use. The logo can be a file in any common image format, including GIF, JPG, PNG, or SVG, and is constrained to a `max-width` of `200px` and a max-height` of `68px`. Image size must not exceed `1 MB` due to constraints on the `ConfigMap` size.
```
Comment (abhatt-rh) [inline modules/adding-a-custom-logo.adoc:14]: Consider using either a [general reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) or use an [object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) for object names.

Further, about the size values in backticks: if  `max-width` and  `max-height` are parameters/specs, it seems valid to have these and their respective values in backticks; not sure if we need backticks for the 1 MB size limit, though.

```suggestion
* Create a file of the logo that you want to use. The logo can be a file in any common image format, including GIF, JPG, PNG, or SVG, and is constrained to a `max-width` of `200px` and a `max-height` of `68px`. Image size must not exceed 1 MB due to constraints on the `ConfigMap` object size.
```

**Review by abhatt-rh — COMMENTED** [review]
Hi @opayne1, Mostly LGTM, just left a comment related to formatting.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**